content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md

```diff
@@ -0,0 +1,291 @@
+### Importing External Files using Webpack Loaders: The Beginners Guide 
+In this article,  we will learn about Webpack and how to use Webpack loaders to import external files while making use of Webpack. 
+
+Webpack may not be straighforward at first, but you will get the hang of it after working with it for some time. 
```

Comment:
You could remove this line. It wouldn't be right introducing a new concept this way